index.html

```diff
@@ -0,0 +1,168 @@
+                       </a>
+                   </li>
+                   <li class="nav__item">
+                       <a href="#" class="nav__link">
+                           Echipamente
+                       </a>
+                   </li>
+                   <li class="nav__item">
+                        <a href="#" class="nav__link">
+                            <form class="nav__item--form">
+                                <input class="nav__item--input" type="search">
+                                <button name="submit" class="nav__item--button">
+                                    <span class="fas fa-search"></span>
+                                </button>
+                            </form>
+                        </a>
+                </li>
+                   <li class="nav__item">
+                       <a href="#" class="nav__link--button">
+                           Oferte
+                       </a>
+                   </li>
+               </ul>
+           </nav>
+       </header>
+       <main class="main">
+           <article class="main__article article">
+               <header class="article__header">
+                   <h1 class="article__title">Titlu blog post</h1>
+                   <span class="article__meta">Scris de Tudor Popa în <time datetime="2021-03-29">29.03.2021</time></span>
+               </header>
+               <div class="article__content">
+                   <h2>Heading 2</h2>
+                   <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tenetur amet labore sapiente unde pariatur reiciendis provident doloremque assumenda beatae aspernatur omnis nulla in saepe porro similique ipsam inventore, neque cumque?. Lorem ipsum dolor sit amet consectetur adipisicing elit. <a href="#">Link example</a>. Quae, facere nobis blanditiis ullam eligendi amet recusandae iste ab praesentium vitae dolorum quibusdam corrupti magni saepe similique atque rem tempora impedit.</p>
+                   <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum perspiciatis nam, aliquid magnam perferendis excepturi ad, doloribus dolorem iure minima dolor molestias. Nihil impedit, vel facilis repudiandae enim doloremque ex.. Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores quaerat tempore possimus nam! Dignissimos, in esse? Deserunt inventore ex totam vero repudiandae. Consectetur saepe ex, cupiditate earum animi reprehenderit dolores.</p>
+                   <blockquote class="article__content--blockq">
+                       <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque doloremque aperiam explicabo voluptatem dicta. Similique aut illo, at nulla totam iusto sapiente dicta, nisi expedita placeat optio. Id, recusandae nostrum. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum nam fugit dicta laboriosam nostrum explicabo sunt adipisci.</p>
+                       <cite class="blockquote__cite">
+                           <p>"</p>

```

Comment:
Asta se adauga prin css pseudo elements

https://learn-the-web.algonquindesign.ca/topics/before-after/